tests/page-objects/default/enketo/death-report.page.js

```diff
@@ -0,0 +1,66 @@
+const moment = require('moment');
+const genericForm = require('./generic-form.wdio.page');
+
+const PLACE_OF_DEATH = {healthFacility: 'health_facility', home: 'home', other: 'other'};
+
+const FORM = 'form[data-form-id="death_report"]';
+const SUMMARY_SECTION = `${FORM} section[name="/death_report/group_review"]`;
+
+const deathDate = () => $(`${FORM} section[name="/death_report/death_details"] input.ignore.input-small`);
+const deathPlace = (value) => $(`${FORM} input[name="/death_report/death_details/place_of_death"][value="${value}"]`);
+const deathInformation = () => $(`${FORM} textarea[name="/death_report/death_details/death_information"]`);
+const patientNameSummary = () => $(`${SUMMARY_SECTION} ` +
+  `span[data-itext-id="/death_report/group_review/r_patient_details:label"] ` +
```

Comment:
If you end up removing the template strings (${..}) thing, then you can use regular single quotes.